src/vs/workbench/browser/parts/editor/textEditor.ts

```diff
@@ -156,12 +155,13 @@ export abstract class AbstractTextEditor<T extends IEditorViewState> extends Abs
 	}
 
 	protected getConfigurationOverrides(): ICodeEditorOptions {
+		const config = this.textResourceConfigurationService.getValue(this.getActiveResource(), 'problems.visibility');
```

Comment:
This method is being called from `computeConfiguration` which already did a lookup of the resource related config, so we should rather pass it down and not look it up again.

Reply:
it specifies `this.textResourceConfigurationService.getValue<IEditorConfiguration>(resource);` which accesses the existing `EditorOptions` - I think I'll still need the lookup since its a different setting not in `EditorOptions`

Reply:
@justschen no, thats just a way to cast to a specific set of settings, but its still returning the same object

Reply:
I see that now yeah - I'll give that a look in the next PR (tracking debt here https://github.com/microsoft/vscode/issues/199246)

Reply:
I am preparing a PR now